issuetracker/pivotaltracker/pivotaltracker.go

```diff
@@ -0,0 +1,43 @@
+package pivotaltracker
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/preslavmihaylov/todocheck/common"
+	"github.com/preslavmihaylov/todocheck/issuetracker"
+)
+
+// IssueTracker implementation for integrating with public & private redmine issue trackers
```

Comment:
All docstrings in this file mention `redmine` instead of `pivotal tracker`.

Reply:
True, I missed this one.

I changed it in the latest commit.